functions/scripts/generate-packing-slips.js

```diff
@@ -0,0 +1,192 @@
+const fs = require('fs');
+const { Readable, finished } = require('stream');
+
+const _ = require('lodash');
+const markdownpdf = require('markdown-pdf');
+const pdfmerge = require('easy-pdf-merge');
+const util = require('util');
+
+const {
+  BULK_ORDER_TABLE,
+  INTAKE_TABLE,
+  ITEMS_BY_HOUSEHOLD_SIZE_TABLE,
+  getAllRecords,
+  getRecordsWithStatus,
+  getBulkOrder,
+} = require('../airtable');
+
+class PackingSlip {
+  constructor(intakeRecord, requested, provided) {
+    this.intakeRecord = intakeRecord;
+    this.requested = requested;
+    this.provided = provided;
+  }
+
+  getMarkdown(itemToCategory) {
+    const fields = this.intakeRecord[1];
+
+    let markdown = `# **${fields.ticketID}**: ${fields.requestName} (${fields.nearestIntersection})\n\n`;
+
+    _.forEach(
+      _.toPairs(
+        _.groupBy(
+          _.toPairs(this.provided),
+          ([item,]) => { return itemToCategory[item]; },
+        )
+      ),
+      ([category, provided]) => {
+        markdown += `### **${category}**:\n`;
+        _.forEach(
+          provided,
+          ([item, numProvided]) => {
+            markdown += `\t- ${numProvided} ${item}\n`;
```

Comment:
Maybe remove the tab to avoid drawing a box around each group?
```suggestion
            markdown += `- ${numProvided} ${item}\n`;
```